### PR TITLE
Improved readability of the communication README

### PR DESCRIPTION
**Pull request by geekygirldawn**
Made several relatively minor changes to improve the readability and organization of the Communication README.

### CODE REVIEW

File: communication/README.md [hunk 1]
```diff
@@ -78,7 +81,7 @@ please propose a specific date on the [Kubernetes Community Meeting Agenda].
 
 ## Conferences
 
-Kubernetes is the main focus of CloudNativeCon/KubeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events] pages.
+Kubernetes is the main focus of KubeCon + CloudNativeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events] pages.
```
Comment (nikhita) [inline communication/README.md]: We should also add China here.

Comment (geekygirldawn) [reply communication/README.md]: Good idea! I just added it. Unfortunately, I also managed to accidentally add a merge commit when I was syncing my fork. As you can see, I also tried to revert it, which I think made a mess of things. I'd appreciate any suggestions for fixing this?

**Comment by geekygirldawn** [comment]
Unfortunately, while adding the China event, I somehow managed to accidentally add a merge commit when I was syncing my fork. I also tried to revert it, as you can see, which I think is where I made a mess of things. I'd appreciate any suggestions for fixing this? Or should I delete it and start over?

**Comment by idealhack** [comment]
@geekygirldawn Hi Dawn, I think you can squash these commits and force push after the review is done.

**Comment by geekygirldawn** [comment]
OK, I had to revert my revert, since the original revert was reverting changes in files that I didn't intend to edit. The commit history looks a bit ugly, but it has the intended result of only making the changes that I intended to make in the communication README.md file.

**Comment by nikhita** [comment]
@geekygirldawn No problem!

@idealhack or @cblecker -- can one of you add the `tide/squash` label? Thanks!

**Comment by nikhita** [comment]
/lgtm

Thanks, Dawn! :smile:

**Pull request merged**